### PR TITLE
Add ./run scripts

### LINKED ISSUE #3: Update boilerplate theme
- Add gulp jekyll
- Add new run tooling

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Add ./run scripts for consistency

## QA

Ensure this theme is set up correctly by runnign through the various scripts.

Fixes #3

### CODE REVIEW

File: .docker-project [hunk 1]
```diff
@@ -0,0 +1 @@
+4b580d2a
```
Comment (nottrobin) [inline .docker-project]: This shouldn't be committed - `.docker-project` should be added to `.gitignore`

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 /build
 /node_modules
++.docker-project
```
Comment (nottrobin) [inline .gitignore]: What do the pluses mean? I suspect this should be simply `.docker-project`

Comment (nottrobin) [inline .gitignore]: We should also add `yarn-error.log`

File: run [hunk 3]
```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# Define the versions of the two docker images
+yarn_image="ubuntudesign/yarn:v0.1.0"
```
Comment (nottrobin) [inline run]: This should now be `canonicalwebteam/yarn:v0.2.0`

File: .gitignore (hunk 2 above)
Comment (barrymcgee) [inline .gitignore]: The plus was erroneously included in a copy/paste, will correct. Are you aware you removed `yarn-error.log` in your `vanilla-docs-theme` [merge](https://github.com/vanilla-framework/vanilla-docs-theme/pull/30/files#diff-a084b794bc0759e7a6b77810e01874f2L7)?

Comment (nottrobin) [inline .gitignore]: I didn't, did I? I removed `yarn.lock`, which is correct, `yarn.lock` should be committed. `yarn-error.log` shouldn't. But you're right, `yarn-error.log` should also be ignored over in `vanilla-docs-theme`

Comment (barrymcgee) [inline .gitignore]: Sorry - I got mixed up log/lock - you're right.

**Comment by barrymcgee** [comment]
@nottrobin Ready for re-review

File: .docker-project (hunk 1 above)
Comment (nottrobin) [inline .docker-project]: This file is still here - it needs to be removed.

**Comment by barrymcgee** [comment]
@nottrobin Now removed...

**Pull request merged**